src/Tools/dotnet-trace/CommandLine/Options/CommonOptions.cs

```diff
@@ -11,28 +11,29 @@ internal static class CommonOptions
     {
         public static Option ProcessIdOption() =>
             new Option(
-                new[] { "-pid" },
-                "The unique identifier of the associated process to connect to.",
-                new Argument<int> { Name = "ProcessId" });
+                aliases: new[] { "-p", "--process-id" },
+                description: "The process to collect the trace from",
+                argument: new Argument<int> { Name = "pid" });
 
         public static Option OutputPathOption() =>
             new Option(
-                new[] { "-o", "--output" },
-                @"The .netperf file name to log events to.",
-                new Argument<string>(defaultValue: $"eventpipe-{DateTime.Now:yyyyMMdd_HHmmss}.netperf") {
-                    Name = "FILE_NAME",
-                });
+                aliases: new[] { "-o", "--output" },
+                description: "The output path for the collected trace data. If not specified it defaults to 'trace.netperf'",
+                argument: new Argument<string>(defaultValue: $"trace.netperf") { Name = "trace-file-path" });
 
-        public static Option CircularBufferOption() =>
+        public static Option ProvidersOption() =>
             new Option(
-                new[] { "--buffersize" },
-                @"Sets the size of the in-memory circular buffer in megabytes.",
-                new Argument<uint>(defaultValue: 64) { Name = "SIZE" });
+                alias: "--providers",
+                description: @"A list of EventPipe providers to be enabled. This is in the form 'Provider[,Provider]', where Provider is in the form: '(GUID|KnownProviderName)[:Flags[:Level][:KeyValueArgs]]', and KeyValueArgs is in the form: '[key1=value1][;key2=value2]'",
+                argument: new Argument<string> { Name = "list-of-comma-separated-providers" }); // TODO: Can we specify an actual type?
 
-        public static Option ProvidersOption() =>
+        // This is a hidden option, currently not in the design-doc spec.
+        private static uint DefaultCircularBufferSizeInMB => 64;
+        public static Option CircularBufferOption() =>
             new Option(
-                aliases: new[] { "--providers" },
-                description: @"A list EventPipe provider to be enabled in the form 'Provider[,Provider]', where Provider is in the form: '(GUID|KnownProviderName)[:Flags[:Level][:KeyValueArgs]]', and KeyValueArgs is in the form: '[key1=value1][;key2=value2]'",
-                argument: new Argument<string> { Name = "PROVIDERS" }); // TODO: Can we specify an actual type?
+                alias: "--buffersize",
```

Comment:
Not in this PR, but if you think these arguments should stick around (and FWIW think it would be a fine idea) you should make an update to the spec so that the documented behavior matches the implementation behavior. This gives a chance for us to ping PMs and other partners/folks in the community that might want to weigh in. For minor stuff we haven't been getting much feedback, but I still think its worth giving everyone that opportunity.

Reply:
I'll update the docs in next PR :)